### PR TITLE
add Open Source Project Template to codebases in incubation

### PR DESCRIPTION
**Pull request by ericherman**
Request additional reviews from @janderssonse and @danneleaf - please suggest edits.

### CODE REVIEW

**Comment by danneleaf** [comment]
I think it looks good ! 👍

**Review by Ainali — APPROVED** [review]

File: codebases/index.html [hunk 1]
```diff
@@ -48,6 +48,16 @@ <h1 class="title">Stage 2: <strong>Incubation</strong></h1>
 
   <div class="cb-listings">
 
+    <div class="cb-listing">
+      <!-- div class="cb-logo"><img src="/photos/codebases/open-source-project-template/open-source-project-template.png" alt="Open Source Project Template logo"></div -->
+      <div class="cb-info">
+        <div class="cb-name">Open Source Project Template</div>
+        <div class="cb-desc">Open Source project helper and get-going template(s)</div>
```
Comment (ElenaFdR) [inline codebases/index.html:55]: 'Getting started' is more conventional than 'get-going'. Was there a reason you chose this?
```suggestion
        <div class="cb-desc">Open Source project helper and getting started template(s)</div>
```

Comment (ericherman) [reply codebases/index.html:55]: This is the short-text description from the github repository.

We can certainly suggest changes to them.

File: codebases/open-source-project-template.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      <h1>Open Source Project Template</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s2">Stage 2: <strong>Incubation</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>A template project containing 'Community Health Files' and relevant checklists.</p>
+      <p>It's intention is to be a practical starter when releasing a project as Open Source.</p>
```
Comment (ElenaFdR) [inline codebases/open-source-project-template.html:29]:
```suggestion
      <p>Its intention is to be a practical starting configuration when releasing a project as Open Source.</p>
```

File: codebases/open-source-project-template.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping launch the Open Source Project Template development by:</p>
+    <ul>
+      <li>Providing feedback on the contents</li>
```
Comment (ElenaFdR) [inline codebases/open-source-project-template.html:57]:
```suggestion
      <li>Providing feedback on the content</li>
```

File: codebases/open-source-project-template.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Community</h3>
+    <dl>
+      <dt>Created by:</dt>
+      <dd><a href="https://www.digg.se" target="_blank">Digg's Open Source Guild</a> and asom (arbetsgruppen för samordning av mjukvara, which translates to working group for coordination of software)</dd>
```
Comment (ElenaFdR) [inline codebases/open-source-project-template.html:84]: I tried to make sure to make sure there was a country context for other codebases.
```suggestion
      <dd><a href="https://www.digg.se" target="_blank">Digg's Open Source Guild</a> and asom (arbetsgruppen för samordning av mjukvara, which translates to working group for coordination of software), Sweden</dd>
```

File: codebases/open-source-project-template.html (hunk 3 above)
Comment (ericherman) [reply codebases/open-source-project-template.html:57]: To me, "contents" seems correct, as the templates and other files in the repository are specific and can be listed with the `ls` command, like a table of contents ... we're not talking about something abstract or amorphous like "web content".

File: codebases/open-source-project-template.html (hunk 2 above)
Comment (ericherman) [reply codebases/open-source-project-template.html:29]: Good catch.

https://github.com/diggsweden/open-source-project-template/compare/main...ericherman:digg-open-source-project-template:spelling-20231023T074123Z?expand=1

**Comment by janderssonse** [comment]
@LGTM , also viewed by @danneleaf  and @ayko1595,

Note: the github-repo description was changed from "get going" to getting started, as suggested.

**Review by Ainali — APPROVED** [review]

**Pull request merged**